03/app.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import MenuItem from "./src/components/MenuItem";
+
+const root = document.querySelector("#root");
+
+// Test
+const test = { text: "Testowy link", url: "https://www.example.com" };
+
+const App = () => {
+    return (
+        <div>
+            <MenuItem item={test} />
```

Comment:
Zwróć uwagę, że ten komponent przyjmuje 2 propsy tj. `text` oraz `url` - więc, aby to działało trzeba zrobić tak: `<MenuItem iext={test.text} url={test/url} />`